content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+Eviction is the process by which {{<glossary_tooltip text="pods" term_id="pod" length="short">}}
+in a {{<glossary_tooltip text="node" term_id="node" length="short">}} are
+terminated. 
+
+In Kubernetes, there are two types of pod eviction:
+
+  1. Graceful eviction
+  2. Out-of-resource eviction ({{<glossary_tooltip text="kubelet" term_id="kubelet" length="short">}}
+     eviction)
+
+{{<caution>}}
```

Comment:
This should be a `{{< note >}}` callout rather than a `{{< caution >}}`; we reserve `{{< caution >}}` for things that are a bit more serious (eg: potential for security problems, but only if you take specific steps). 

There is also `{{< warning >}}` for things you really ought to know (eg: don't use an allow-all authorization configuration except for local testing)

Reply:
Removed the note entirely.